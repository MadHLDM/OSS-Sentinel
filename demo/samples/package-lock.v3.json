{
  "name": "demo",
  "lockfileVersion": 3,
  "packages": {
    "": {
      "name": "demo",
      "version": "1.0.0"
    },
    "node_modules/express": {
      "version": "4.18.2"
    },
    "node_modules/lodash": {
      "version": "4.17.20"
    }
  }
}

