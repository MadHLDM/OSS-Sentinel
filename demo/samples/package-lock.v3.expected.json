[
  { "name": "express", "version": "4.18.2", "ecosystem": "npm" },
  { "name": "lodash",  "version": "4.17.20", "ecosystem": "npm" }
]

